Aliucord/src/main/java/com/aliucord/coreplugins/rn/Patches.kt

```diff
@@ -4,9 +4,12 @@
  * Licensed under the Open Software License version 3.0
  */
 
+@file:Suppress("MISSING_DEPENDENCY_SUPERCLASS")
+
```

Comment:
can u remove

Reply:
But it gets rid of the error for those funny classes
I'll remove if it's a must